std/file.d

```diff
@@ -388,6 +388,34 @@ Returns: Array of characters read.
 
 Throws: $(D FileException) on file error, $(D UTFException) on UTF
 decoding error.
+
+---
+import std.file;
+import std.utf;
+
+void main(string[] args)
+{
+    try
+    {
+        // Read and validate UTF-8 file.
+        auto utf8Data  = readText("test.txt");
+
+        // Read and validate UTF-16 file.
+        auto utf16Data = readText!(wstring)("test.txt");
+
+        // Read and validate UTF-32 file.
+        auto utf32Data = readText!(dstring)("test.txt");
+    }
+    catch (UTFException ex)
+    {
+        // Handle validation errors
+    }
+    catch (FileException ex)
+    {
+        // Handle errors
+    }
+}
+---
```

Comment:
This example looks like it will work as-is in a unittest block (though it wouldn't be testing much other than that it compiles).